src/PolicyEditor.js

```diff
@@ -496,6 +535,116 @@ export class PolicyEditor extends Component {
+                                        onChange={this.handleChange}
+                                        value={stateProperty(this, "policy.actions.beforeSnapshotRoot.mode")}>
+                                        <option value="essential">essential</option>
+                                        <option value="optional">optional</option>
+                                        <option value="async">async</option>
+                                    </Form.Control>
+                                </WideValueColumn>
+                                {EffectiveValue(this, "actions.beforeSnapshotRoot.mode")}
+                            </Row>
+                            <hr />
+                            <Row>
+                                <LabelColumn name="After Snapshot" help="Script to run after snapshot." />
+                                <WideValueColumn>{OptionalFieldNoLabel(this, "", "policy.actions.afterSnapshotRoot.path", {})}</WideValueColumn>
+                                {EffectiveValue(this, "actions.afterSnapshotRoot.path")}
+                            </Row>
+                            <Row>
+                                <LabelColumn name="Timeout" help="Timeout in seconds before Kopia kills the process." />
+                                <WideValueColumn>{OptionalNumberField(this, "", "policy.actions.afterSnapshotRoot.timeout", {})}</WideValueColumn>
+                                {EffectiveValue(this, "actions.afterSnapshotRoot.timeout")}
+                            </Row>
+                            <Row>
+                                <LabelColumn name="Command Mode" help="essential (must succeed, default behavior), optional (failures are tolerated) or async (kopia will start the action but not wait for it to finish)." />
+                                <WideValueColumn>
+                                    <Form.Control as="select" size="sm"
+                                        name="policy.actions.afterSnapshotRoot.mode"
+                                        onChange={this.handleChange}
+                                        value={stateProperty(this, "policy.actions.afterSnapshotRoot.mode")}>
+                                        <option value="essential">essential</option>
+                                        <option value="optional">optional</option>
+                                        <option value="async">async</option>
+                                    </Form.Control>
+                                </WideValueColumn>
+                                {EffectiveValue(this, "actions.afterSnapshotRoot.mode")}
+                            </Row>
+                        </Accordion.Body>
+                    </Accordion.Item>
+                    <Accordion.Item eventKey="folder-actions">
+                        <Accordion.Header><FontAwesomeIcon icon={faFileAlt} />&nbsp;Folder Actions</Accordion.Header>
```

Comment:
can we do another icon? - this one is used for Logging - perhaps use "gears" and "folder-gear" instead: https://fontawesome.com/v6.0/icons/gears?s=thin

Reply:
It seems you're using v4 - there's only cog and cogs there, used those for now